scripts/list-repos-matching-extensions.js

```diff
@@ -0,0 +1,46 @@
+const { searchByCriteria } = require("../util/github");
+
+module.exports = {
+    description: "list language usage within an organization",
+    options: {
+        organization: {
+            alias: "o",
+            demandOption: true,
+            describe: "the GitHub organization to search for language usage",
+            type: "string"
+        },
+        extensions: {
+            alias: "e",
+            demandOption: true,
+            describe: "a comma seperated list of extensions being searched for",
+            type: "string"
+        },
+    },
+    action: async (octokit, argv) => {
+        const reposMatchingExtensions = {};
+
+        const extensionsArray = argv.extensions.split(",")
+        const formattedExtensionsString = "+extension:" + extensionsArray.join("+extension:")
+
+        const queryResults = await searchByCriteria(octokit, `*+org:${argv.organization}${formattedExtensionsString}`)
+
+        for (const result of queryResults) {
+            const repo_full_name = result.repository.full_name
```

Comment:
let's use camelcase for variable names

Reply:
I'll get our eslint config added to repo so we don't have to worry about comments like these in the future 😄